src/style.css

```diff
@@ -0,0 +1,60 @@
+body {
+  margin: 0;
+  font-family: Roboto;
```

Comment:
Which family will be displaying if google fonts not responding?
Add additional families like `Arial` or `sens-serif`.